src/demo/DlDialogBoxDemo.vue

```diff
@@ -6,6 +6,7 @@
         <dl-dialog-box
             ref="modalOne"
             v-model="isOpenedFirstModal"
+            :draggable="true"
```

Comment:
Add a switch here ontop of the button to control if its draggable or not for the demo